extensions/mssql/src/dashboard/serviceEndpoints.ts

```diff
@@ -8,46 +8,69 @@ import * as azdata from 'azdata';
 import * as nls from 'vscode-nls';
 const localize = nls.loadMessageBundle();
 
-import * as Utils from '../utils';
+import * as utils from '../utils';
+
+const mgmtProxyName = 'mgmtproxy';
+const grafanaEndpointName = 'metricsui';
+const grafanaDescription = localize('grafana', "Metrics Dashboard");
+const logsuiEndpointName = 'logsui';
+const kibanaDescription = localize('kibana', "Log Search Dashboard");
+const sparkHistoryEndpointName = 'spark-history';
+const sparkHistoryDescription = localize('sparkHistory', "Spark Jobs Management and Monitoring Dashboard");
+const yarnUiEndpointName = 'yarn-ui';
```

Comment:
We were going to use the description field returned right? Although the old CTP bits don't have that so I suppose if we're supporting that we still need this